packages/map/src/map/hooks.jsx

```diff
@@ -64,14 +74,22 @@ export const useMapInstance = (nodeRef, options) => {
       setMap(map);
     };
 
+    const handleBoundsChange = (e) => {
+      const isCausedByUser = Boolean(e.originalEvent);
+      if (!isCausedByUser) return;
+
+      onBoundsChange(map.getBounds().toArray());
```

Comment:
Maybe pass bounds instead of array? There can be cases where you will need bounds

Reply:
I thought about it but `bounds` is an object with many methods and I wouldn't want to have our apps rely on implementation details of this package. Let's say we change the underlying map library, then we would need to keep supporting the mapbox `bounds` object api.